### ISSUE TITLE
[Enhancement] e621 API / blacklist import

### ISSUE THREAD

**Issue by Lycraon** [PawCorp/walltaker#48]
<img width="663" alt="image" src="https://user-images.githubusercontent.com/80824508/170825809-6163bc86-6e01-4a6a-acb1-abf3c8d2afe4.png">

**Comment by pupgray** [comment]
Put as low priority, this requires quite a bit of effort (E621 does authentication via an API key, not oauth, so I have to get an API key from users, store it, and run E621 api calls with their api) for not much gain (user's premade blacklist is always applied)

**Comment by LittleFopsy** [comment]
> Put as low priority, this requires quite a bit of effort (E621 does authentication via an API key, not oauth, so I have to get an API key from users, store it, and run E621 api calls with their api) for not much gain (user's premade blacklist is always applied, cub posts)

I agree that it's not much gain in regard of using the user's blacklist from e621 as you actually could just this up per link. Maybe the blacklist of e6 is a little bit more advanced. For unblocking of cub posts I would  suggest just creating an account and using the linked key for all requests made on walltaker. This should be way more easier to implement and if someone doesn't want to see something like this they can add it to their link's blacklist.

That are my thoughts of this at least.
